resolwe/flow/management/commands/list_docker_images.py

```diff
@@ -0,0 +1,61 @@
+""".. Ignore pydocstyle D400.
+
+==================
+List Docker images
+==================
+
+"""
+
+import functools
+import operator
+
+import yaml
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from resolwe.flow.models import Process
+
+
+class Command(BaseCommand):
+    """List Docker images used by processes."""
+
+    help = "List Docker images used by processes in either plain text or YAML (for Ansible)"
+
+    def add_arguments(self, parser):
+        """Add an argument to specify output format."""
+        parser.add_argument('--format', dest='format', default='plain',
+                            help="Set output format ('plain' [default] or 'yaml')")
+
+    def handle(self, *args, **options):
+        """Handle command list_docker_images."""
+        # Check that the specified output format is valid
+        if options['format'] != 'plain' and options['format'] != 'yaml':
+            raise CommandError("Unknown output format: %s" % options['format'])
+
+        # Gather a list of all custom Docker images the processes are using
+        unique_docker_images = set(
+            p.requirements['executor']['docker']['image']
```

Comment:
`image` is not the required field in `docker` dict, so this has to be checked separately.